docs/dev/GCP-setup.md

```diff
@@ -0,0 +1,101 @@
+# GCP SetUp for JRS Testing
+# **Table of Contents**
+
+- [Pre set up](#pre-setup)
+- [Initial set up](#initial-setup)
+- [Gcloud set up](#gcloud-setup)
+- [Personal JSON set up for regression](#json-setup)
+- [Creating instances manually](#manual-creation)
+- [Deleting instance manually](#delete-manually)
+
+<a name="pre-setup"></a>
+
+# **Pre set up**
+1. Follow directions in e-mail `Welcome to your new Google Account for Hashgraph` from `google workspace team`. You will need either a physical security or google to use your cell phone for 2FA.
+2. Send your public key you are planning to use to GCP admin in order to make sure you can access all machines.
+
+<a name="initial-setup"></a>
+
+# **Initial set up**
+1. Go to `console.cloud.google.com`.
+2. Login with your `hedera.com` email address to use.
+3. Agree to the terms. 
+   
+![terms-agreement.png](../assets/terms-agreement.png)
+
+4. Click on `Select a Project` on the top most menu. 
+   
+![select-project.png](../assets/select-project.png)
+
+5. Select `Swirlds.com -> Hedera Regression`. If you do not see `Hedera Regression` click on the “ALL” tab and follow the folders down till you find `Hedera Regression`.
+
+![choose-from-folders.png](../assets/choose-from-folders.png)
+
+<a name="gcloud-setup"></a>
+
+# **Gcloud set up**
+To run regression from your computer you will need to install gcloud on your computer
+
+1. Follow the instructions to install the sdk for your OS at [LINK](https://cloud.google.com/sdk/docs/install).
+2. After successful installation, open terminal or command prompt and  run `gcloud init`.
+3. When asked to login agree: `Y`.
+4. It provides a link to login. Follow the link in browser (make sure you are logged into browser with your hedera.com gmail account).
+    - Click Allow, if `Google Cloud SDK wants to access the Google Account`. 
+    - Copy Code you are given.
+    - Paste into the waiting field on command prompt.
+
+![copy-code.png](../assets/copy-code.png)
+
+![link-in-browser.png](../assets/link-in-browser.png)
+
+5. Pick the cloud project to use as `hedera-regression`.
+6. Configure default compute region, by selecting `Y` and pick `us-east1-c`.
```

Comment:
Picking a compute region closer to the user would be more appropriate(in most cases).

Reply:
@abhishek-hedera  Agreed. But not all regions have the same quota. If the quota is less for a region , instances might not be created successfully.